### ISSUE TITLE
prompt window boxline bug with chinese charactor

### ISSUE THREAD

**Issue by guazai22** [helix-editor/helix#2442]
### Summary

line 306

[
![2022-05-09-201101_1440x900](https://user-images.githubusercontent.com/64355495/167408096-3da2104f-5303-4d31-a4f3-e4cc4532a481.PNG)
](url)

### Platform

linux

### Terminal Emulator

kitty

### Helix Version

helix 22.03 (d4e45fd4)

**Comment by the-mikedavis** [comment]
This may be a rendering bug in Kitty. See also https://github.com/helix-editor/helix/issues/2416

**Comment by archseer** [comment]
You can't fit a full-width character in a regular monospace character's box, so the two cells overlap. How that's rendered then depends on the implementation

**Comment by kirawi** [comment]
Closing as it seems to be specific to the terminal.

**Comment by pickfire** [comment]
No, this is not specific to terminal. I can reproduce in konsole as well.
